README.md

```diff
@@ -275,3 +260,192 @@ datetime
 2018-06-04 00:00:00+00:00  0.07947329054545459  602.5116327272724
 
 ```
+
+### Exchange funds flow
+
+Fetch the difference between the tokens that were put in minus the tokens that were taken out from an exchange for a given slug in the selected time period.
```

Comment:
Let's use the words `deposited` and `withdrawn` as these clearly indicate what we are talking about.